frontend/www/js/omegaup/course/edit.js

```diff
@@ -264,6 +268,7 @@ OmegaUp.on('ready', function() {
                 .catch(function(error) {
                   UI.apiError(error);
                   assignmentDetails.show = true;
+                  assignmentDetails.columnError = error.errorcolumn;
```

Comment:
```suggestion
                  assignmentDetails.columnError = error.errorcolumn || '';
```

de otra forma se podría estar asignando `undefined` a `assignmentDetails.columnError` y eso no es legal.

Reply:
Listo